website/content/docs/ui/custom-messages/index.mdx

```diff
@@ -0,0 +1,88 @@
+---
+layout: docs
+page_title: Manage custom messages
+description: >-
+  Use custom messages in the Vault UI to share system-wide alerts.
+---
+
+# Manage custom messages in the Vault UI
+
+Use custom banners and modals in the Vault UI to share system-wide alerts for all Vault UI users. Users can then dismiss messages by clicking "X" on a banner or "Confirm" for a modal.
+![Vault UI sidebar. The "Custom Messages" button in Settings section is outlined with a red box](/img/custom-messages-sidebar.png)
+
+## Before you start
+
+- **You must have Vault 1.16.0 or higher installed**.
+- **You must have admin-level permission for the Vault UI**.
+
+## Add a Custom Message
+
+<Note title="Messages are inheritable">
+  Child namespaces inherit all messages created on the parent namespace.
+</Note>
+<Note title="Messages have a maximum limit of 100">
+  Vault can only store up to 100 messages.
+</Note>
```

Comment:
@schavis I'm not sure if multiple notes is too much information for the user, should we make the notes into bullet points instead?

We want to inform them that:
1. Messages are inheritable.
2. There can only be a maximum of 100 messages created. If it exceeds more than 100 message, they will need to delete one message before creating another.
3. It is recommended to avoid having more than one active modal at once since it can be intrusive for users.

Could you help with the language of these notes?

Reply:
Generally, multiple notes are bad because it trains folks to visually skip over them. Since this is general info about messages and not specific to creating them, I suggest moving the information about inheritability and size limit out of the aside boxes and making it part of the introduction to the page.

Reply:
I added a suggestion for new text in the intro :)